### ISSUE TITLE
[Enhancement] Monster Wishlist

### ISSUE THREAD

**Issue by HeeHoFrosty** [madcowfred/padherder-issues#285]
Would it be possible to add a Monster wishlist that works similar to the My Material page? For example: when you put a monster in the wishlist, the list will show which dungeons you can get them from and alerts you if the monster is only available in an event dungeon.

**Comment by pknintendodude** [comment]
I agree with this-- One of the problems I'm having is I want to make a team, but I don't have the required monster. Making a fake entry is good and all, but it would be nice to have a sort of "My Wishlist" page that keeps track of mons (and final form; not their level or evo mats) for the purposes of Team building. That way, I'm building towards a team without having to remember what subs I wanted to run.

As for the dungeon alerts, I think you could probably get away with no Descendeds. If someone wants to run a Descended Dungeon, they probably won't forget that it's going to be up soon, as they'll be planning for it long in advance. As for popular but hard to get lines, definitely. This covers Mystic Knight Line, Healer Girl Line. Special dungeons are fine for this one (Eg if someone wants Dryad, it would say "Ancient Wood Dragon - expert/etc up for x days..." but not "1.5x Drop Rate on Sacred Mt. Bakkes")

**Comment by madcowfred** [comment]
This has some things in common with GH-312 and GH-205, tagging so I remember.

**Comment by ummdude** [comment]
I'd like this + the notifications of dungeons that can drop this monster having higher drop rates/stamina reduction.
